tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -10,6 +10,7 @@
 from iib.workers.tasks.utils import RequestConfigMerge
 
 
+@pytest.mark.parametrize('source_fbc, target_fbc', [(False, False), (False, True), (True, True)])
```

Comment:
What about the case `(True, False)` when source == FBC and target != FBC?

Reply:
Already covered by https://github.com/release-engineering/iib/blob/50c78e92eb983c9fd3db76276c49c6b7b50cdfad/tests/test_workers/test_tasks/test_build_merge_index_image.py#L608

Reply:
Tks!